README.md

```diff
@@ -114,6 +114,18 @@ period via the --process-timeout flag. E.g.,
 nosetests --processes=8 --process-timeout=120 tests/test_binaries.py
 ```
 
+### Redis
+
+If you'd like to use redis for state serialization (instead of disk), install
+redis using your host package manager, then run
+
+```
+pip install manticore[redis]
```

Comment:
The reader interested in this section really should be appending `[redis]` to whichever `pip install` command of the above three they chose. Could we change the language to reflect this to avoid people blindly doing `pip install` even if they did `pip install --user` above, for example? I still like keeping the code block to show an example of how this looks, since it's somewhat uncommon i think.